.travis.yml

```diff
@@ -27,3 +28,7 @@ matrix:
   exclude:
     - rvm: 2.4.7
       gemfile: gemfiles/rails_6_0.gemfile
+    - rvm: 2.7.1
+      gemfile: gemfiles/rails_5_0.gemfile
+    - rvm: 2.7.1
+      gemfile: gemfiles/rails_4_2.gemfile
```

Comment:
why would we want a rails 4.2 ? did you mean `rails_5_2`

Reply:
We don't want rails 4.2, which is why we're excluding this permutation in the matrix.